### PR TITLE
Add bug reporting system for development

### PR DESCRIPTION
**Pull request by Nebrethar**

### CODE REVIEW

**Comment by Nebrethar** [comment]
This is not the final product (9/19/2018) for the requested changes.
Just making some progress and removing the button.

File: src/background.js [hunk 1]
```diff
@@ -69,6 +69,27 @@ async function main() {
         }
     }
 
+    async function bug(e) {
+        let [URL] =  await browser.tabs.query({active: true, currentWindow: true});
+        let domain = (URL.url).split("/")[2];
```
Comment (zombie) [inline src/background.js]: It's never a good idea to parse URLs manually, and a convenient `URL` helper is available:
```
    let [tab] = await browser.tabs.query({active: true, currentWindow: true});
    let domain = new URL(tab.url).host;
```

File: src/background.js [hunk 2]
```diff
@@ -69,6 +69,27 @@ async function main() {
...
+        let body = "I found a bug in Global Consent Manager!";
+        body += "%0A%0A";
+        body += "(Explain the issue here)";
+        await browser.tabs.create({
+            url: "https://github.com/SociallyCompute/Global-Consent-Manager/issues/new?title="
+                + "Bug found while browsing " + domain + "&body=" + body +
+                "&projects=loading..." + "&labels=bug",
+        });
+        window.close();
```
Comment (zombie) [inline src/background.js]: This is not needed.

**Comment by zombie** [comment]
Err, not sure why those comments got hidden, but here are direct links:
https://github.com/SociallyCompute/Global-Consent-Manager/pull/24#discussion_r219585461
https://github.com/SociallyCompute/Global-Consent-Manager/pull/24#discussion_r219585478

**Comment by Nebrethar** [comment]
I think you may have reviewed an older commit. 

Currently, function `bug(e)` in background.js references an HTML document and creates a tab for it.
I think I had removed that `window.close()` by now

**Pull request merged**